### PR TITLE
Bug 1384577 – Ensure the clipboard bar is shown for long enough.

### PR DESCRIPTION
**Pull request by jhugman**
This ups this from 4.0 seconds to 4.5 seconds.

https://bugzilla.mozilla.org/show_bug.cgi?id=1384577

### CODE REVIEW

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -2259,14 +2259,14 @@ extension BrowserViewController: TabManagerDelegate {
...
-    func show(buttonToast: ButtonToast, afterWaiting delay: Double = 0, duration: Double = SimpleToastUX.ToastDismissAfter) {
+    func show(buttonToast: ButtonToast, afterWaiting delay: DispatchTimeInterval = SimpleToastUX.ToastDelayBefore, duration: DispatchTimeInterval = SimpleToastUX.ToastDismissAfter) {
         // If BVC isnt visible hold on to this toast until viewDidAppear
         if self.view.window == nil {
             self.pendingToast = buttonToast
             return
         }
 
-        let time = DispatchTime(uptimeNanoseconds: DispatchTime.now().uptimeNanoseconds) + Double(Int64(delay * Double(NSEC_PER_SEC))) / Double(NSEC_PER_SEC)
+        let time = DispatchTime(uptimeNanoseconds: DispatchTime.now().uptimeNanoseconds) + delay
```
Comment (garvankeeley) [inline Client/Frontend/Browser/BrowserViewController.swift]: nit: maybe `DisplatchTime.now() + delay` instead?

File: Client/Frontend/Browser/ButtonToast.swift [hunk 2]
```diff
@@ -144,14 +144,14 @@ class ButtonToast: UIView {
...
+    func showToast(duration: DispatchTimeInterval = SimpleToastUX.ToastDismissAfter) {
         layoutIfNeeded()
         UIView.animate(withDuration: SimpleToastUX.ToastAnimationDuration, animations: {
                 self.animationConstraint?.update(offset: 0)
                 self.layoutIfNeeded()
             },
             completion: { finished in
-                let dispatchTime = DispatchTime.now() + Double(Int64(duration * Double(NSEC_PER_SEC))) / Double(NSEC_PER_SEC)
+                let dispatchTime = DispatchTime.now() + duration
                 DispatchQueue.main.asyncAfter(deadline: dispatchTime, execute: {
```
Comment (garvankeeley) [inline Client/Frontend/Browser/ButtonToast.swift]: nit: I prefer the compact syntax of:
```
DispatchQueue.main.asyncAfter(deadline: .now() + duration) {
   self.dismiss(false)
}```
```

Comment (jhugman) [inline Client/Frontend/Browser/ButtonToast.swift]: :+1:

File: Client/Frontend/Browser/BrowserViewController.swift (hunk 1 above)
Comment (jhugman) [inline Client/Frontend/Browser/BrowserViewController.swift]: :+1:

**Pull request merged**